pkg/sdk/logging/api/v1beta1/fluentd_types.go

```diff
@@ -147,7 +147,9 @@ type FluentdDrainConfig struct {
 	Enabled bool `json:"enabled,omitempty"`
 	// Container image to use for the drain watch sidecar
 	Annotations map[string]string `json:"annotations,omitempty"`
-	Image       ImageSpec         `json:"image,omitempty"`
+	// Delete PVC on drain success
+	DeleteVolume bool      `json:"delete,omitempty"`
```

Comment:
maybe we should be more explicit about what we're deleting
```suggestion
	DeleteVolume bool      `json:"deleteVolume,omitempty"`
```

Reply:
Was the comment related to the typo or would you prefer something else than `deleteVolume`? 
My first thought was `deleteVolumeOnDrainSuccess`. Tell me if you have any ideas.

Reply:
It was related to the typo only I didn't know it was a typo 😄 I think `deleteVolume` along with the documentation (i.e. comment that you wrote) should be fine 👍